src/Person.jsx

```diff
@@ -0,0 +1,36 @@
+import './App.scss';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried,
+    partnerName = 'I am not married' } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        My name is
+        {' '}
+        {name}
```

Comment:
Consider using [Template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) for better readability